index.html

```diff
@@ -20,482 +20,9 @@
-                      type="email"
-                      name="user-email"
-                      class="form-input"
-                      required
-                    />
-                  </label>
-                  <label class="form_field">
-                    <span class="hidepass">
-                      <span class="input_text">Password</span>
-                      <span class="hidepass_icon">
-                        <img
-                          src="./images/atoms/a-icon-hidepass.svg"
-                          alt="icon"
-                        />
-                      </span>
-                      <span class="hidepass_text">show</span>
-                    </span>
-                    <input
-                      type="password"
-                      name="user-password"
-                      class="form-input"
-                      required
-                    />
-                  </label>
-                  <button type="submit" class="button-dark form_contact-us_btn">
-                    Send message
-                  </button>
-                  <p class="form_text">
-                    If you need to have a DNA first, just contact us at
-                    <a href="mailto:email@gmail.com">email@gmail.com</a>
-                  </p>
-                </form>
-                <img src="./images/map.png" alt="map" width="380" />
-              </div>
-            </div>
-          </div>
-        </div>
-      </section>
-    </main>
-    <footer class="footer">
-      <div class="container">
-        <div class="footer_box">
-          <ul class="social_list">
-            <li class="social_item">
-              <a
-                href="https://www.facebook.com/"
-                target="_blank"
-                class="social_link"
-              >
-                <div class="social_icon">
-                  <img src="./images/atoms/a-icon-facebook.svg" alt="" />
-                </div>
-              </a>
-            </li>
-            <li class="social_item">
-              <a
-                href="https://www.instagram.com/"
-                target="_blank"
-                class="social_link"
-              >
-                <div class="social_icon">
-                  <img src="./images/atoms/a-icon-instagram.svg" alt="" />
-                </div>
-              </a>
-            </li>
-            <li class="social_item">
-              <a
-                href="https://dribbble.com/"
-                target="_blank"
-                class="social_link"
-              >
-                <div class="social_icon">
-                  <img src="./images/atoms/a-icon-dribbble.svg" alt="" />
-                </div>
-              </a>
-            </li>
-          </ul>
-          <a href="" class="logo">BlogWorld</a>
-          <p class="footer_box_text">&copy;2019 All Rights Reserved.</p>
-        </div>
-      </div>
-    </footer>
+    <div id="root"></div>
+    <script src="data.json"></script>
```

Comment:
instead of including `data.json` file in to html, and creating global variable, please import it directly to the file where it is used
![Screenshot 2021-09-22 at 11 20 30](https://user-images.githubusercontent.com/33399163/134308300-d269cf52-9d93-4241-9ec5-df5d81d66bb1.png)